apps/checkout/src/sections/Addresses/useUserAddressSelect.tsx

```diff
@@ -0,0 +1,46 @@
+import { AddressFragment, AddressTypeEnum } from "@/graphql";
+import { useCheckout } from "@/hooks/useCheckout";
+import { getById } from "@/lib/utils";
+import { useEffect, useState } from "react";
+import { isMatchingAddress } from "./utils";
+
+export interface UseUserAddressSelectProps {
+  defaultAddress?: Pick<AddressFragment, "id"> | null;
```

Comment:
Seems like `defaultAddressId` would make more sense here instead of an object with single key

Reply:
Done